### PR TITLE
make Controller Reference Migrations directly with SQL

### PR DESCRIPTION
**Pull request by dpfaffenbauer**

### CODE REVIEW

File: bundles/CoreBundle/Command/Migration/ControllerReferenceMigrationCommand.php [hunk 1]
```diff
@@ -84,29 +86,76 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
-        $condition = 'type NOT IN (:types)';
+        $tables = [
+            'documents_page',
+            'documents_snippet',
+            'documents_email',
+            'documents_newsletter',
+            'documents_printpage'
+        ];
+        $queryBuilders = [
+//            'documents_page' => $queryBuilder
```
Comment (weisswurstkanone) [inline bundles/CoreBundle/Command/Migration/ControllerReferenceMigrationCommand.php:97]: still needed ?

**Comment by blankse** [comment]
@dpfaffenbauer Works for me and is very fast 👍🏼

**Comment by weisswurstkanone** [comment]
@blankse , thanks for testing!

File: bundles/CoreBundle/Command/Migration/ControllerReferenceMigrationCommand.php (hunk 1 above)
Comment (dpfaffenbauer) [reply bundles/CoreBundle/Command/Migration/ControllerReferenceMigrationCommand.php]: removed it

**Comment by blankse** [comment]
@dpfaffenbauer Is somewhere a limit? It is not all controller migrated. When I call it again it will migrated...

**Comment by dpfaffenbauer** [comment]
no limit... could be cache though

**Comment by blankse** [comment]
@dpfaffenbauer After clearing the data cache in the backend all is fine. Can you trigger this in the command?

**Comment by weisswurstkanone** [comment]
@dpfaffenbauer 🏓

**Comment by dpfaffenbauer** [comment]
done

**Comment by weisswurstkanone** [comment]
thx!

**Pull request merged**